packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -149,8 +149,14 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
     const nativeProps = getNativeProps<React.HTMLAttributes<HTMLSpanElement>>(this.props, htmlElementProperties, ['className']);
 
     return (
-      <div className={this._classNames.text} id={this.state.labelId}>
-        <span className={this._classNames.innerText} role="status" aria-live={this._getAnnouncementPriority()} {...nativeProps}>
+      <div className={this._classNames.text}>
+        <span
+          className={this._classNames.innerText}
+          id={this.state.labelId}
```

Comment:
I'm curious, what's the reason for moving the id prop? That has the potential to break people's tests or other assumptions.

Reply:
yeah, good point. I think i can move it back